Backport of valgrind-3.7.0-automake-1.11.2.patch

Retain the old behavior of the build system even with automake 1.11.3.

Reported-by: Oleg Bulatov <dmage@bk.ru>
X-Gentoo-Bug: 397429
X-Gentoo-Bug: 403779
X-Gentoo-Bug-URL: https://bugs.gentoo.org/show_bug.cgi?id=397429
X-Gentoo-Bug-URL: https://bugs.gentoo.org/show_bug.cgi?id=403779
X-KDE-Bug: 290719
X-KDE-Bug-URL: https://bugs.kde.org/show_bug.cgi?id=290719
Signed-off-by: Anthony G. Basile <blueness@gentoo.org>

diff -Naur valgrind-3.6.1.orig//Makefile.am valgrind-3.6.1/Makefile.am
--- valgrind-3.6.1.orig//Makefile.am	2010-10-20 16:19:45.000000000 -0400
+++ valgrind-3.6.1/Makefile.am	2012-02-16 20:27:23.000000000 -0500
@@ -57,7 +57,8 @@
 # default.supp, as it is built from the base .supp files at compile-time.
 dist_noinst_DATA = $(SUPP_FILES)
 
-pkglib_DATA = default.supp
+vgdatadir = $(pkglibdir)
+vgdata_DATA = default.supp
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = valgrind.pc
